src/app/functions/server/getExperiments.test.ts

```diff
@@ -100,9 +100,6 @@ describe("getExperiments", () => {
 
   it("calls Cirrus V2 when feature flag is enabled with preview param", async () => {
```

Comment:
comment(blocking): please update the wording of this test, because it's no longer calling/testing a flag